internal/mock/mocks.go

```diff
@@ -0,0 +1,219 @@
+	CorruptedSigEnv = strings.Replace(ValidSigEnv, "0fX0", "1fX0=", 1)
+)
```

Comment:
I wanted this mock to be available for all of notation-go code as there are few packages that would benefit, like `signature`, `verification` and `plugin`. Do you still see any issues since this is an internal package?

Reply:
I mean we could save this JSON file in the `testdata` folder like `internal/mock/testdata/envolope.json`.